docs/documentation/guides/python.mdx

```diff
@@ -38,7 +38,7 @@ python3 -m venv env
 source env/bin/activate
 ```
 
-Install Flask and [infisical-python](https://github.com/Infisical/infisical-python), the client Python SDK for Infisical.
+Install Flask and [infisical-python](https://infisical.com/docs/sdks/languages/python), the client Python SDK for Infisical.
```

Comment:
New SDKs do not use service tokens so this guide doesn't make sense any more since it starts with `Create an Infisical Token`. I think you should leave all the sdk links as is because guides will need to be updated accordingly. Simple link updates will make things more confusing